action_server/.gitignore

```diff
@@ -0,0 +1 @@
+*.swp
```

Comment:
Move to `.gitignore` in main folder of the repo?

Reply:
See 05430a5743e1ad00c23fd8ba128ffb5a33ddeb0a